<launch>

    <!-- You should edit these parameters to point to your topics -->

    <arg name="camera_topic" default="/usb_cam/image_raw" doc="Camera topic name" />
    <arg name="cmd_vel_pub" default="/nav/cmd_vel" doc="The topic that cmd_vel messages will be published on when replaying a map (can be the same as cmd_vel_sub)" />
    <arg name="cmd_vel_sub" default="/husky_velocity_controller/cmd_vel" doc="The topic that cmd_vel messages will be recorded from when making a map (can be the same as cmd_vel_pub)" />
    <arg name="odom_topic" default="/husky_velocity_controller/odom" doc="The topic that odometry information will be read from" />
    <arg name="additional_record_topics" default="" doc="Use this parameter to record any additional topics you wish to be in your map and repeated each traversal. Space separated topics names." />

    <!-- You should not need to edit anything below this line. -->

    <group ns="bearnav2">

        <node pkg="bearnav2" type="representations-ros-1.py" name="representations" respawn="true" output="screen" args="-OO" >
            <param name="camera_topic" type="string" value="$(arg camera_topic)" />
        </node>

    </group>
</launch>
